kibana/saved-objects/kibana-8-x-auth-and-session.ndjson

```diff
@@ -1,35 +1,34 @@
+{"excludedObjects":[],"excludedObjectsCount":0,"exportedCount":33,"missingRefCount":0,"missingReferences":[]}
```

Comment:
shall we add a new line.

Reply:
Fixed with https://github.com/wso2-extensions/identity-elk-integration/pull/14